templates/p5_env.bt

```diff
@@ -64,35 +64,35 @@ typedef struct
 
 typedef struct
 {
-    SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field02<name = "Field Model Diffuse Red">;
-	f32 Field06<name = "Field Model Diffuse Green">;
-	f32 Field0A<name = "Field Model Diffuse Blue">;
-	f32 Field0E<name = "Field Model Diffuse Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
 	f32 Field12<name = "Field Model Ambient Red">;
 	f32 Field16<name = "Field Model Ambient Green">;
 	f32 Field1A<name = "Field Model Ambient Blue">;
 	f32 Field1E<name = "Field Model Ambient Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field22<name = "Field Model Specular Red">;
-	f32 Field26<name = "Field Model Specular Green">;
-	f32 Field2A<name = "Field Model Specular Blue">;
-	f32 Field2E<name = "Field Model Specular Alpha">;
+	f32 Field22<name = "Field Model Diffuse Red">;
+	f32 Field26<name = "Field Model Diffuse Green">;
+	f32 Field2A<name = "Field Model Diffuse Blue">;
+	f32 Field2E<name = "Field Model Diffuse Alpha">;
+    SetBackColor( MyRandom( 0xFFFFFFFF ) );
+	f32 Field32<name = "Field Model Specular Red">;
+	f32 Field36<name = "Field Model Specular Green">;
+	f32 Field3A<name = "Field Model Specular Blue">;
+	f32 Field3E<name = "Field Model Specular Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field32<name = "Field Model Emissive Red">;
-	f32 Field36<name = "Field Model Emissive Green">;
-	f32 Field3A<name = "Field Model Emissive Blue">;
-	f32 Field3E<name = "Field Model Emissive Alpha">;
+	f32 Field42<name = "Field Model Emissive Red">;
+	f32 Field46<name = "Field Model Emissive Green">;
+	f32 Field4A<name = "Field Model Emissive Blue">;
+	f32 Field4E<name = "Field Model Emissive Alpha">;
 	SetBackColor( MyRandom( 0xFFFFFFFF ) );
-    f32 Field42;
-	f32 Field46;
-	f32 Field4A;
-	f32 Field4E;
+    f32 Field52<name = "Field52">;
+	f32 Field56<name = "Field56">;
+	f32 Field5A<name = "Field5A">;
+	f32 Field5E<name = "Field5E">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field52<name = "Field Model Light X (Unsure)">;
-	f32 Field56<name = "Field Model Light Y (Unsure)">;
-	f32 Field5A<name = "Field Model Light Z (Unsure)">;
+	f32 Field62<name = "Field Model Light X">;
```

Comment:
I know this is a general issue with this file, but could I ask you to update the field names to match the name in the name tag? Just for the fields you added would be a good start, I imagine doing all of them is too much work.

Reply:
No problem, do you have a preferred naming style (FieldModelLightX or Field_Model_Light_X. or fieldModelLightx) ?